Game Coders/flappy_bird/flappy_bird.pde

```diff
@@ -1,13 +1,19 @@
 Bird b;
+
 PVector gravity = new PVector(0, 0.5);
 
 void setup(){
-  size(400,800);
+  size(480,800);
```

Comment:
```suggestion
  size(400,800);
```
I suppose for this scenario, `size(400,800)` is a standard option, what do you think? is there any particular motivation to change it? 😄

Reply:
Hey sorry I was experimenting with the values!

Reply:
That is really nice, and nothing to be sorry about 🚀